internal/project/framework/dbdrivers/redis.go

```diff
@@ -1,10 +1,11 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package dbdrivers
 
 import (
 	"encoding/json"
 
-	"{{ .PkgPath }}/framework/internals/aes"
+	/**#bean*/
+	"demo/framework/internals/aes" /*#bean.replace("{{ .PkgPath }}/framework/internals/aes")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines